### PR TITLE
Fix SSAO 2 detection.

### PR DESCRIPTION
**Pull request by sebavan**

### CODE REVIEW

File: src/PostProcesses/RenderPipeline/Pipelines/ssao2RenderingPipeline.ts [hunk 1]
```diff
@@ -155,7 +155,7 @@ export class SSAO2RenderingPipeline extends PostProcessRenderPipeline {
         if (!engine) {
             return false;
         }
-        return engine.getCaps().drawBuffersExtension;
+        return engine.webGLVersion >= 2;
```
Comment (deltakosh) [inline src/PostProcesses/RenderPipeline/Pipelines/ssao2RenderingPipeline.ts:158]: This does not exist in webgl1?

Comment (sebavan) [reply src/PostProcesses/RenderPipeline/Pipelines/ssao2RenderingPipeline.ts:158]: Nope the shaders are not supported and use dedicated syntax from glsl 3.0. I ll check in a bit if it is possible to fix easily.

Comment (sebavan) [reply src/PostProcesses/RenderPipeline/Pipelines/ssao2RenderingPipeline.ts:158]: Basically relying on arrays initialization, modulo and others operators.

Comment (deltakosh) [reply src/PostProcesses/RenderPipeline/Pipelines/ssao2RenderingPipeline.ts:158]: not a problem though. No need to fix the shaders honestly

**Pull request merged**